spec/models/conference_spec.rb

```diff
@@ -7,6 +7,22 @@
   it { is_expected.to have_many(:attendees) }
   it { is_expected.to validate_presence_of(:name) }
   it { is_expected.to validate_presence_of(:round) }
+  it { is_expected.to validate_presence_of(:starts_on) }
+  it { is_expected.to validate_presence_of(:ends_on) }
+
+  describe 'it checks the chronological order of dates' do
+    subject { FactoryGirl.build(:conference) }
```

Comment:
It's a common best practise and in my opinion also increases readability to leave a blank line between `subject` and the first example.

I'm also in general not sure about the use of `subject` here? I'm not working with Rails on a day to day basis, but from what I remember, the `errors` field is only appended to a model and updated accordingly if `model.valid?` is called... So I'm not sure if your expectation here actually does what it should.